refomat_lad_table.R

```diff
@@ -0,0 +1,23 @@
+lad_raw <- read_csv(here::here("analysis", "RiverLAD.csv"))
```

Comment:
can you please add the google cloud pull or source some doc that pulls in this csv.

Reply:
added google cloud pull